test/unit/specs/components/SpnDescentPlots.spec.js

```diff
@@ -6,4 +6,9 @@ describe('SnpDescentPlots', () => {
       expect(SnpDescentPlots.name).to.equal('snp-descent-plot')
     })
   })
+  describe('forEachLine', () => {
+    it('should', () => {
```

Comment:
test is missing description

Reply:
Last commit was not pushed yet, this test is gone now